spec/helpers/commodity_helper_spec.rb

```diff
@@ -188,4 +188,68 @@
       expect(helper.applicable_additional_codes).to eq(expected_options)
     end
   end
+
+  describe '#applicable_measure_units' do
+    let(:expected_measure_units) do
+      {
+        'DTN' => {
+          'abbreviation' => '100 kg',
+          'measure_sids' => [20_005_920, 20_056_507, 20_073_335, 20_076_779, 20_090_066, 20_105_690, 20_078_066, 20_102_998, 20_108_866, 20_085_014],
+          'measurement_unit_code' => 'DTN',
+          'measurement_unit_qualifier_code' => '',
+          'unit' => 'x 100 kg',
+          'unit_hint' => 'Enter the value in decitonnes (100kg)',
+          'unit_question' => 'What is the weight of the goods you will be importing?',
+        },
+      }
+    end
+
+    context 'when deltas are applicable' do
+      let(:user_session) { build(:user_session, :with_commodity_information, :deltas_applicable) }
+
+      it 'fetches the uk commodity' do
+        helper.applicable_measure_units
+
+        expect(Api::Commodity).to have_received(:build).with('uk', anything, anything)
```

Comment:
Is `anything` this a reserved keyword?

Reply:
It instantiates a matcher of the type RSpec::Mocks::ArgumentMatchers::AnyArgMatcher. Basically it's a method call that returns a AnyArgMatcher which makes any argument of any type pass comparison